docs/benchmarks/data-comparisson.md

```diff
@@ -64,6 +64,10 @@ CLASSIFICATIONS. #
 
 **Other files:**<br />The `lisa.rel`file is an old version of the relevance assessments which helps to understand new version, but is not considered important to use. 
 
+**Parsing problems**:<br />
+* documents: they are all in seperate files, the newline as online marker to seperate ID, title and text can be a bit tricky
```

Comment:
Actual review: online marker shoul be "only marker"?

Reply:
Oh yeah of course, I will change that